tests/integration/targets/kernel_blacklist/tasks/main.yml

```
@@ -0,0 +1,90 @@
+---
+- name: set destination filename
+  set_fact:
+    bl_file: '{{ output_dir }}/blacklist-ansible.conf'
```

```suggestion
    bl_file: '{{ remote_tmp_dir }}/blacklist-ansible.conf'
```
`output_dir` is on the controller side, not on the remote side. (Keeping this will break once split-controller testing is merged in ansible-core.) Also you need to make the tests depend on `setup_remote_tmp_dir` in meta/main.yml.

---

I copied from ansible_galaxy_install's and I used the same thing in interfaces_file (that's been just merged).

---

> I copied it from ansible_galaxy_install's

I fixed these 10 days ago: https://github.com/ansible-collections/community.general/commits/main/tests/integration/targets/ansible_galaxy_install

> and I used the same thing in interfaces_file (that's been just merged). Will need to fix those as well.

Hmm, I guess I overlooked that there. Can you please fix them?

---

Sure. By the way, in the first one I copied it from somewhere else. A quick search brings dozens of cases.

---

All should be fixed (except the ones re-added after my Improve CI PR), or at least the ones which run on the remote part in the split-controller tests. Otherwise #3345 wouldn't have passed :)